.gitignore

```diff
@@ -1,6 +1,3 @@
-bundle
-cjs
-esm
```

Comment:
In order to use the git repository as npm package, we have to upload the transpiled javascipt files.